scss/modules/label.scss

```diff
@@ -1,35 +1,42 @@
 .husky-label-error,
 .husky-label-warning,
 .husky-label-success {
-    $height: 70px;
-
     width: 100%;
-    height: $height;
     position: relative;
     top: 0;
     left: 0;
     color: $white;
-    padding-left: 55px;
     font-size: $smallFontSize;
     z-index: 200;
 
+    &.big {
+        height: 70px;
```

Comment:
Another solution would be to do something like `&-big` and use `@extend .husky-label-warning` and only adapt the height there. Then we would only adapt the height there. But it might be harder to understand, and makes it maybe a little bit harder to change the size of the label dynamically, in case that will ever be a requirement.